### PR TITLE
adding cluster status request to admin client

### PR DESCRIPTION
**Pull request by barnjamin**

### CODE REVIEW

**Comment by barnjamin** [comment]
Signed the contrib agreement, added a sanity check function.  Is this a reasonable pattern to follow for the rest of the admin like functions?  I intend to add at least a couple more like assigning/compacting regions.

File: integration_test.go [hunk 1]
```diff
@@ -29,6 +29,23 @@ var host = flag.String("host", "localhost", "The location where HBase is running
 
 const table = "test1"
 
+// ClusterStatus gets a summary of relevant cluster stats
+func ClusterStats(client gohbase.AdminClient) error {
```
Comment (Timoha) [inline integration_test.go]: Could you please make it a full test, something like this:
```
func TestClusterStatus(t *testing.T) {
      ac := gohbase.NewAdminClient(*host)
      defer ac.Close()
      stats, err := ac.ClusterStatus()
      if err != nil {
          t.Fatal(err)
      }
      if len(stats.GetMaster().GetHostName()) == 0 {
           t.Fatal("Master hostname is empty in ClusterStatus")
      }
}
```

**Comment by Timoha** [comment]
Yes, this looks great, add whatever you need as it's likely that others would need it as well.
Note that travis builds are failing for your change (linting like forgot to add comments to exported functions).

**Comment by barnjamin** [comment]
Sorry for the spam

**Comment by barnjamin** [comment]
Closing for #53

**Pull request closed without merging**